services/provider/server/server.go

```diff
@@ -24,6 +25,8 @@ import (
 
 const (
 	TicketAnnotation = "ocs.openshift.io/provider-onboarding-ticket"
+
+	SecretMountPoint = "/mnt/cert"
```

Comment:
This is imported in other packages, can we call it `ProviderCertsMountPoint`?

Reply:
yeah